api/disabled_parameters.go

```diff
@@ -7,8 +7,103 @@ import (
 
 	"github.com/getkin/kin-openapi/openapi3"
 	"github.com/labstack/echo/v4"
+	"gopkg.in/yaml.v3"
 )
 
+// DisplayDisabledMap is a struct that contains the necessary information
+// to output the current config to the screen
+type DisplayDisabledMap struct {
+	// A complicated map but necessary to output the correct YAML.
+	// This is supposed to represent a data structure with a similar YAML
+	// representation:
+	//  /v2/accounts/{account-id}:
+	//    required:
+	//      - account-id : enabled
+	//  /v2/accounts:
+	//    optional:
+	//      - auth-addr : enabled
+	//      - next: disabled
+	Data map[string]map[string][]map[string]string
+}
```

Comment:
But The type here seems very similar to what we already have for DisableMap. So there might be a way to avoid this similar but different type. This would become pretty useful when it comes time to parse the config file because you would use `yaml.Unmarshal` to do most of the work.

I think it's OK to leave it like this now, but maybe consider it again when you start parsing the file.